### ISSUE TITLE
Questions about the classification feature

### ISSUE THREAD

**Issue by Melon-Xu** [2U1/Qwen2-VL-Finetune#171]
Thank you so much for this valuable feature! I have adapted it to my video classification task. Here I have several questions:
1. During the classification finetuning, I noticed that LLM is frozen. Will it lower the performance of the classification?
2. For setting the CLASS_2_ID, assume that I have 3 classes, do I need to modify it accordingly?
3. My dataset does not contain validation set, could I set off the validation part?

Thank you again for this valuable feature!
